dot-base/keycloak
# feat: update keycloak to use quarkus keycloak version 21-1-2

# Ticket 🎫

This closes
- https://github.com/dot-base/keycloak/issues/64
- https://github.com/dot-base/keycloak/issues/65
- https://github.com/dot-base/keycloak/issues/69

Related PRs:
Changes in keycloak deployment: https://github.com/dot-base/deployments/pull/121

Keycloak config needs changes on all instances before releasing dotbase-23-4: https://github.com/dot-base/keycloak/issues/70


# Description 📖

* updates Keycloak to version  21.1 with quay.io base image
* Realm import on start only if no realm is existing
* Custom TOTP endpoints

# How to Test? 🧪

A test image based on the changes of this PR was built: [ghcr.io/dot-base/keycloak:PR-66](ghcr.io/dot-base/keycloak:PR-66
)

This image is deployed on [staging](staging.dotbase.charite.de) with the current dotbase 23-3 stack. See also: https://github.com/dot-base/deployments/pull/121

Steps to test manually can be found in #65 .

# Checklist ✅
